### PR TITLE
fix(DB): Bael'dun Digsite mob flags fixed

### PR DESCRIPTION
**Pull request by neifion-00000000**
Issue: Mobs at Bael'dun Digsite were incorrectly flagging players for pvp.
Faction now matches other ally friendly horde quest NPCs.

## Changes Proposed:
-  Mob entries 2989 and 2990 faction changed to 122 to resolve pvp flagging from Mulgore questing.  This keeps them friendly with ally without flagging horde players PVP.

## Tests Performed:
- Working on my test server. 

## How to Test the Changes:

1. on either faction character  .go c 26602
2. observe they are not pvp flagged in their icon, on ally char they will be friendly on horde char they will be hostile

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1632513740571144600.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1632513740571144600');
+UPDATE `acore_world`.`creature_template` SET `faction`='122' WHERE  `entry`=2990;
+UPDATE `acore_world`.`creature_template` SET `faction`='122' WHERE  `entry`=2989;
```
Comment (Helias) [inline data/sql/updates/pending_db_world/rev_1632513740571144600.sql:3]:
```suggestion
UPDATE `acore_world`.`creature_template` SET `faction`=122 WHERE  `entry` IN (2990, 2989);

```

**Review by Helias — APPROVED** [review]

**Comment by Asthalor** [comment]
Tested, **all good**.

**Before**

Horde
8069 before

Alliance

8069 2 before

**After**

Horde

8069 2 after

Alliance

8069 after

**Comment by Hacki95** [comment]
What is the source for the change? Could it be that they were like this on retail?

**Comment by neifion-00000000** [comment]
> What is the source for the change? Could it be that they were like this on retail?

there were no forced pvp quests in the leveling to 60 in classic, aside from that classic as a source currently has it in the suggested corrected state

**Pull request merged**